Dockerfile

```
@@ -0,0 +1,33 @@
+FROM python:3.12.2-slim
+
+RUN ["apt-get", "update"]
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)